Build failure when included in TS project

### What version of Hono are you using?

4.11.4

### What runtime/platform is your app running on? (with version if possible)

node + npm or pnpm 

### What steps can reproduce the bug?

See https://github.com/michielbdejong/node-typescript-boilerplate/pull/1 for a minimal example where I include hono in an otherwise (almost) empty project, and both `pnpm build` and `npm run build` start failing. You can see it failing in GitHub Actions on that PR.

### What is the expected behavior?

My TS project that uses hono would build successfully, both on my laptop and in GitHub Actions

### What do you see instead?

```
node_modules/.pnpm/hono@4.11.4/node_modules/hono/dist/types/request.d.ts:19:5 - error TS1166: A computed property name in a class property declaration must have a simple literal type or a 'unique symbol' type.

19     [GET_MATCH_RESULT]: Result<[unknown, RouterRoute]>;
       ~~~~~~~~~~~~~~~~~~
```

### Additional information

_No response_

--- a/src/request/constants.ts
+++ b/src/request/constants.ts
@@ -1,1 +1,1 @@
-export const GET_MATCH_RESULT: symbol = Symbol()
+export const GET_MATCH_RESULT: unique symbol = Symbol()

